{{cookiecutter.repo_name}}/.travis.yml

```diff
@@ -1,36 +1,53 @@
-language: python
-python: '3.5'
 sudo: false
+language: python
+python:
+{% for env in ['2.6', '2.7', '3.3', '3.4', '3.5', 'pypy'] %}
+  - "{{ env }}"
+{% endfor %}
 env:
+{% if cookiecutter.c_extension_support|lower == "yes" %}
   global:
-    LD_PRELOAD=/lib/x86_64-linux-gnu/libSegFault.so
+    - LD_PRELOAD=/lib/x86_64-linux-gnu/libSegFault.so
+    - SEGFAULT_SIGNALS="abrt segv"
+{% endif %}
   matrix:
-    - TOXENV=check
-{% for env in ['2.6', '2.7', '3.3', '3.4', '3.5', 'pypy'] %}
-    - TOXENV={{ env }}-cover
+    - COVERAGE=cover
       {%- if cookiecutter.coveralls|lower == 'yes' %}
         {%- if cookiecutter.c_extension_support|lower == "yes" %},extension-coveralls{% endif %},coveralls
       {%- endif %}
       {%- if cookiecutter.codecov|lower == 'yes' %},codecov{% endif %}
-    - TOXENV={{ env }}-nocov
-{% endfor %}
+    - COVERAGE=nocov
+matrix:
+  include:
+    - python: "3.4"
+      env: TOXENV=check
+
 before_install:
-  - python --version
-  - uname -a
-  - lsb_release -a
+  - easy_install --version
 install:
   - pip install tox
+    {%- if cookiecutter.test_runner|lower == "pytest" %} pytest-travis-fold{% endif %}
   - virtualenv --version
-  - easy_install --version
-  - pip --version
   - tox --version
+
+before_script:
+{% if cookiecutter.test_runner|lower == "pytest" %}
+  - export TOX_TESTENV_PASSENV=TRAVIS
```

Comment:
The [corresponding](https://bitbucket.org/hpk42/tox/src/f6cca79ba7f6522893ab720e1a5d09ab38fd3543/tox/config.py#config.py-399) code performs roughly the following:
  1. Take default variables: `"PATH", "PIP_INDEX_URL", "LANG", "LD_LIBRARY_PATH"`
  2. Add these from `TOX_TESTENV_PASSENV`
  3. Add all existing environment variables matching `passenv` from `tox.ini`.

So everything should be fine. I added this on purpose, in case if user decides to remove `passenv = *` from `tox.ini`.